modbus_tk/modbus.py

```diff
@@ -177,6 +182,28 @@ def execute(
                 # slave + func + bytcodeLen + bytecode x 2 + crc1 + crc2
                 expected_length = 2 * quantity_of_x + 5
 
+        elif function_code == defines.READ_FILE_RECORD:
+            is_read_function = True 
+            if (isinstance(number_file, tuple)
+                    and isinstance(starting_address, tuple)
+                    and isinstance(quantity_of_x, tuple)
+                    and len(number_file)==len(starting_address)==len(quantity_of_x)>0
+            ):
+                count_seq=len(number_file)
+            else:
+                raise ModbusInvalidRequestError(
+                    'For function READ_FILE_RECORD param'
+                    'starting_address, quantity_of_x, number_file must be tuple ()'\
+                     'of one length > 0 (by the number of requested sub_seq)'
+                     )
```

Comment:
indentation of closing parenthesis should be at same level than raise

Reply:
Accepted, resolved in the next commits